tests/test_cargo_publish.rs

```diff
@@ -84,7 +176,8 @@ test!(simple {
 });
 
 test!(git_deps {
-    let p = project("foo")
+    let root = paths::root().join("git_deps");
+    let p = repo(&root)
```

Comment:
the test would discover the parent git repo and fail to pass because it would be dirty. I made them repos so they have their own git stack and stop there.